cli/azd/pkg/project/service.go

```diff
@@ -95,3 +96,23 @@ func (svc *Service) Deploy(
 
 	return result, progress
 }
+
+func (svc *Service) getOverriddenEndpoints() []string {
+	overriddenEndpoints := svc.Environment.GetServiceProperty(svc.Config.Name, "ENDPOINTS")
+	if overriddenEndpoints != "" {
+		var endpoints []string
+		err := json.Unmarshal([]byte(overriddenEndpoints), &endpoints)
+		if err != nil {
+			// This can only happen if the environment file is manually edited.
```

Comment:
(Or if the type of the output was not an array, which would be due to an authoring error)  Glad we don't return an error and thanks for the log statement.